checker/watcher.go

```diff
@@ -237,3 +246,12 @@ func (pw *PeersWatcher) Shutdown() error {
 
 	return nil
 }
+
+func shouldConsider(mdInfo *mdInfo) bool {
+	consider := false
+	if mdInfo.ipsecState == "active" &&
```

Comment:
you don't want to consider activating and updating-active states?

Reply:
@alena1108 those are transitioning states right? I did think about those states, but keeping the original problem of the upgrade taking time, thought it's better to exclude those. Do you think otherwise?

Reply:
I would rather exclude the states related to upgrade. And leave the rest around